app/src/main/java/com/cornellappdev/resell/android/ui/components/global/messages/MessageTag.kt

```diff
@@ -0,0 +1,84 @@
+package com.cornellappdev.resell.android.ui.components.global.messages
+
+import androidx.compose.animation.core.animateFloatAsState
+import androidx.compose.animation.core.tween
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.interaction.MutableInteractionSource
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.dp
+import com.cornellappdev.resell.android.ui.theme.Style
+import com.cornellappdev.resell.android.ui.theme.animateResellBrush
+
+@Composable
+fun MessageTag(
+    text: String,
+    modifier: Modifier = Modifier,
+    unreads: Int,
+    active: Boolean = false,
+    onClick: () -> Unit,
+) {
```

Comment:
Same thing here, any reason to make ur own instead of using/modifying `ResellTag`?